src/components/NavBar.jsx

```diff
@@ -1,14 +1,16 @@
 import '../styles/NavBar.css'
+import { NavLink } from 'react-router-dom'
+
 
 export function NavBar() {
 
     return (
         <>
             <nav>
                 <ul>
-                    <li>Coding</li>
-                    <li>Cooking</li>
-                    <li>Football</li>
+                    <NavLink to='/coding' className='nav-link'><li>Coding</li></NavLink>
+                    <NavLink to='/cooking' className='nav-link'><li>Cooking</li></NavLink>
+                    <NavLink to='/football' className='nav-link'><li>Football</li></NavLink>
```

Comment:
Instead of hardcoding the topics ( which makes the site brittle ) it would be ideal to fetch the topics from the api in order to then choose a topic. If the database removed the football topic, the site would then be broken unless you manually updated it in it's current form